harness/determined/common/experimental/experiment.py

```diff
@@ -9,6 +9,7 @@
 from determined.common import api
 from determined.common.api import bindings
 from determined.common.experimental import checkpoint, trial, workspace
+from determined.common.experimental._util import OrderBy
```

Comment:
This violates [style guide](https://google.github.io/styleguide/pyguide.html#22-imports):
> Use import statements for packages and modules only, not for individual classes or functions.

We haven't been enforcing this broadly, but such enforcement is coming (I think there's an existing ticket).

Reply:
discussed offline. imported as such to obfuscate _util. TODO and JIRA added.